utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -712,6 +712,32 @@ int main(int argc, char **argv)
 
                     CHECK(hipHostFree(lutBuffer));
                 }
+                case 35:
+                {
+                    testCaseName = "glitch";
+                    RpptChannelOffsets *rgbOffsets;
+                    CHECK(hipHostMalloc (&rgbOffsets, batchSize * sizeof(RpptChannelOffsets)));
```

Comment:
Would suggest to move this allocation and deallocation outside the loop
You need to check the test case outside followed by allocation and deallocation if test case is glitch

Something similar is done for here. Please refer
https://github.com/r-abishek/rpp/blob/daabb36f7ddbb8b2c5a501baacba2f1938e832ed/utilities/test_suite/HIP/Tensor_hip.cpp#L359
https://github.com/r-abishek/rpp/blob/daabb36f7ddbb8b2c5a501baacba2f1938e832ed/utilities/test_suite/HIP/Tensor_hip.cpp#L347